packages/controller/contracts/ANController.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity 0.5.17;
+
+import "./interfaces/IMiniMeLike.sol";
+import "./interfaces/ITokenController.sol";
+
+
+contract ANTController is ITokenController {
+    string private constant ERROR_NOT_MINTER = "ANC_SENDER_NOT_MINTER";
+
+    IMiniMeLike public ant;
+    address public minter;
+
+    event ChangedMinter(address indexed minter);
+
+    /**
+    * @dev Ensure the msg.sender is the minter
+    */
+    modifier onlyMinter {
+        require(msg.sender == minter, ERROR_NOT_MINTER);
+        _;
+    }
+
+    constructor(IMiniMeLike _ant, address _minter) public {
+        ant = _ant;
+        minter = _minter;
+    }
+
+    function mintANT(address _owner, uint256 _amount) external onlyMinter returns (bool) {
+        return ant.generateTokens(_owner, _amount);
+    }
+
+    function changeMinter(address _newMinter) external onlyMinter {
+        minter = _newMinter;
+        emit ChangedMinter(_newMinter);
+    }
+
+    // Default controller settings for allowing token transfers.
+    // ANT was compiled with solc 0.4.8, so there is no point in marking any of these functions as `view`.
```

Comment:
Since this one is using solc 0.5.x, I think it makes sense to declare these as `view` to avoid having a compiler issue, it doesn't affect how the other contracts were compiled if we are not going to redeploy them

Reply:
Note that there are no compiler warnings (AFAIK) because the interface is declared without `view`.

On the fence about this one too, but thought to keep it like this:

1. The old interface does not declare `constant`
2. ANT wouldn't use a staticcall when calling into these functions anyway

Reply:
True, based on what you mention in (2), I'd keep it as it is then